core/trino-main/src/main/java/io/trino/security/AccessControl.java

```diff
@@ -147,7 +151,9 @@ public interface AccessControl
      * Check if identity is allowed to change the specified schema's user/role.
      *
      * @throws AccessDeniedException if not allowed
+     * @deprecated use {@link #checkCanSetEntityAuthorization}
      */
+    @Deprecated
     void checkCanSetSchemaAuthorization(SecurityContext context, CatalogSchemaName schemaName, TrinoPrincipal principal);
```

Comment:
These can be removed leaving only the new generic entity version

Reply:
Removed.